documents/operating.md

```diff
@@ -11,6 +12,17 @@ Contents:
 6.  Policies
 
 ---------------------------------------------------------------------
+SECTION 0: 📓📓📓 Definitions 📓📓📓
+---------------------------------------------------------------------
+
+* **Bad Idea Factory Limited Liability Company ("the Company")** is a Limited Liability Company domiciled in Pennsylvania.
+* The Company may also be referred to as "**Bad Idea Factory**" or "**BIFFUD**".
+* "**Corporate Overlord**" shall mean an "owner" of the Company under Pennsylvania state law.
+* The "**Board of Directors**" ("the **Board**") of the Company, as defined by Pennsylvania state law, shall consist of all current Corporate Overlords of the Company.
+* "**Consensus**" shall mean TKTKTK.
+* "**Quorum**" shall mean TKTK
```

Comment:
1) the process is literally the process for amending the operating agreement (whatever that will entail in Section 5)

2/3) yes we should still have some quorum. if we decide to actually do time period-based voting and governance, then pass/fail for that should still require some minimum # of COs voting.

i think all three of these points belong in Section 5 (Governance) and what we end up drafting there.

let us not get caught up on the specific definitions before we need them. this section is meant to be here and edited as we flesh out the rest of the document. it is up here because it's good practice (since the rest of the doc may use a turn of phrase many times) and also probably required of us to define things like "corporate overlord" in legal terms